helpers/Physics.h

```diff
@@ -0,0 +1,380 @@
+#ifndef _HELPERS_PHYSICS_H_
+#define _HELPERS_PHYSICS_H_
+
+#include "../include/z64.h"
+#include "Matrix.h"
+
+typedef struct {
+	s32   J; // Amount of strand limbs + 1
+	Vec3f headPos;
+	Vec3s headRot;
+	f32*  lengthList;
+	struct {
+		Vec3f scale;  // Gfx scale
+		u32   dlist;
+		u8    segID;  // For matrix
+		u8    noDraw; /* If there needs to be calculations
+		                 to get in position before drawing */
+	} gfx;
+	struct {
+		s32    numVec;      // num of how many vec are used from vecList
+		Vec3f* vecList;     // "collision" spheres
+		f32    dist;        // XYZ distance from limb to vecList member
+	} spheres;
```

Comment:
```suggestion
	struct {
		s32    num; // amount of spheres
		Vec3f* centers;
		f32    radius;
	} spheres;  // "collision" spheres
```

Reply:
~~I can't tell if these spheres are meant to be collision for the strand being simulated, or represent stuff the strand is supposed to collide with?~~
I figured from the example and code it must be outside collision, to represent eg shoulders, torso, head... neat
I don't think "spheres" is descriptive enough but I don't see a good name for it, "externCollisionSpheres" is kinda too verbose